README.md

```diff
@@ -46,7 +69,11 @@ Using Ruby...
 - Add "Belle" to the `guests` array
 
 ```rb
-# Your code goes here...
+
+residents.delete_at(1)
```

Comment:
You would be able to do this if there was a variable called `residents` that contained an array. In this example, however, `residents` is a key in the `town` hash. So we would need to access it and delete "Belle" like so...

```rb
town[:residents].delete_at(1)
```